docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -163,30 +163,6 @@ Provider's `/config` folder has the same structure of  `/config` folder in CAPI
 - clusterctl-settings.json file
     - if the `configFolder` value is defined, update from `/config` to `/config/default`.
 
-## Upgrade cert-manager to v1.1.0
-
-NB. instructions assumes "Required kustomize changes to have a single manager watching all namespaces and answer to webhook calls"
-should be executed before this changes.
-
-**Changes in the `/config/certmanager` folder:**
-
-1. Edit the `/config/certmanager/certificates.yaml` file and replace all the occurrences of `cert-manager.io/v1alpha2`
-with `cert-manager.io/v1`
-
-**Changes in the `/config/default` folder:**
-
-1. Edit the `/config/default/kustomization.yaml` file and replace all the occurencies of
-   ```
-         kind: Certificate
-         group: cert-manager.io
-         version: v1alpha2
-   ```
-   with
-   ```
-         kind: Certificate
-         group: cert-manager.io
-         version: v1
-   ```
```

Comment:
I don't think we should delete this paragraph because it provides instruction for the Cluster API providers developers about how to change their provider manifest for using cert-manager v1.1.0 objects.

Reply:
Ok, I am reverting it!